api/libproject/environments.rb

```diff
@@ -98,6 +98,11 @@ def env_with_defaults(env, config)
         :dest_cdr_project => "fc-aou-cdr-synth-stable",
         :auth_domain_group_email => "GROUP_all-of-us-registered-stable@firecloud.org",
       },
+      "controlled" => {
```

Comment:
missing prod

Reply:
prod was already here and stable wasn't, not sure why - see L139